app/lib/actions/katello/host/auto_attach_subscriptions.rb

```diff
@@ -10,6 +10,9 @@ def queue
 
         def plan(host)
           action_subject(host)
+          if ::Organization.find_by_id(host.organization_id).simple_content_access?
+            fail ::Katello::HttpErrors::BadRequest, _("Auto attach is disabled in Simple Content Access mode")
```

Comment:
While it makes perfect sense to raise the error here since this action could be called in several places, we have been making efforts for performance to move certain actions out of dynflow.

For this reason, I think we should raise the error wherever this action is called from. That way we won't be filling the Monitor -> Tasks view with a bunch of 'Failed' tasks. That way we are more likely to have successful records in the tasks list